resources/views/database/modifyCity.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends('layouts.master')
+
+@section('body')
+    <div class="container">
+        <div class="row my-3">
+            <a href="{{route("cityView")}}" class="btn btn-primary col-md-2"> <i class="fas fa-arrow-circle-left"></i>
+                Manage </a>
+        </div>
+    </div>
+    <form method="POST" action="{{route('modifyCitySubmit')}}">
+        {{ csrf_field() }}
+        <div class="container">
+            <div class="row mt-3 mb-5">
+                <div class="card h-100 shadow">
+                    <div class="card-header">
+                        <h3>Current City</h3>
+                        <h4>{{$city->cityName}}</h4>
+                    </div>
+                    <div class="card-body">
+                        <div class="d-flex">
+                            <div class="input-group mb-3">
+                                <div class="input-group-prepend">
+                                    <span class="input-group-text" id="inputGroup-sizing-default">New City</span>
```

Comment:
Instead of `New City` could we say `New Name`? I think it would be much clearer for the user - otherwise they might be worried that they are creating a new City entry, rather than editing the name of the current one.